addons/mrp/static/src/widgets/timer.js

```diff
@@ -66,7 +66,8 @@ export class MrpTimer extends Component {
     }
 
     get durationFormatted() {
-        if(this.props.value!=this.state.duration && this.props.record && !this.readonly){
+        if(this.props.value!=this.state.duration && this.props.record && this.props.record.isDirty){
+            if (typeof this.props.setDirty==='function')this.props.setDirty(false);
```

Comment:
`        if(this.props.value != this.state.duration &&  this.props.record?.isDirty){
            if (typeof this.props.setDirty === 'function') {
            this.props.setDirty(false);
}`